package.json

```diff
@@ -10,6 +10,45 @@
     "test-app"
   ],
   "scripts": {
-    "prepare": "cd addon && yarn build"
+    "prepare": "cd addon && yarn build",
+    "release": "release-it"
+  },
+  "devDependencies": {
+    "release-it": "^14.13.1",
+    "release-it-lerna-changelog": "^4.0.1",
+    "release-it-yarn-workspaces": "^2.0.1"
+  },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org"
+  },
+  "release-it": {
+    "plugins": {
+      "release-it-lerna-changelog": {
+        "infile": "CHANGELOG.md",
+        "launchEditor": false
+      },
+      "release-it-yarn-workspaces": {
+        "workspaces": [
+          "addon"
+        ],
+        "additionalManifests": {
+          "dependencyUpdates": [
+            "test-app/package.json"
+          ],
+          "versionUpdates": [
+            "package.json",
+            "test-app/package.json"
+          ]
+        }
+      }
+    },
+    "git": {
+      "tagName": "v${version}"
+    },
+    "github": {
+      "release": true,
+      "tokenRef": "GITHUB_AUTH"
+    },
+    "npm": false
```

Comment:
Why npm is false here? Is it not supposed to publish to npm?

Reply:
@villander correct, this folder/package.json *is not* published to npm as the addon code now lives in `addon` folder

Reply:
Description of this option can be found in https://github.com/release-it/release-it/blob/master/docs/npm.md

https://github.com/rwjblue/release-it-yarn-workspaces#usage has good explanation why this is needed in case of monorepo setup